packages/table/src/table.tsx

```diff
@@ -615,6 +622,32 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
+    private syncScrollPosition(_prevProps: ITableProps, prevState: ITableState) {
```

Comment:
This didn't really need to be a separate function since it's only used in one place, but I prefer boxing related functionality into a helper to reduce cognitive load when reading the parent function.